pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -79,23 +83,31 @@ fun rememberPagerState(
  *
  * In most cases, this will be created via [rememberPagerState].
  *
+ * The `offscreenLimit` param defines the number of pages that
+ * should be retained on either side of the current page. Pages beyond this limit will be
+ * recreated as needed. This value defaults to `1`, but can be increased to enable pre-loading
+ * of more content.
+ *
  * @param pageCount the initial value for [PagerState.pageCount]
  * @param currentPage the initial value for [PagerState.currentPage]
  * @param currentPageOffset the initial value for [PagerState.currentPageOffset]
+ * @param offscreenLimit the number of pages that should be retained on either side of the
+ * current page. This value is required to be `1` or greater.
  */
 @ExperimentalPagerApi
 @Stable
 class PagerState(
     @IntRange(from = 0) pageCount: Int,
     @IntRange(from = 0) currentPage: Int = 0,
     @FloatRange(from = 0.0, to = 1.0) currentPageOffset: Float = 0f,
+    val offscreenLimit: Int = 1,
 ) : ScrollableState {
     private var _pageCount by mutableStateOf(pageCount)
     private var _currentPage by mutableStateOf(currentPage)
     private var _currentLayoutPageOffset by mutableStateOf(currentPageOffset)
 
-    internal var currentLayoutPage by mutableStateOf(currentPage)
-        private set
+    internal val layoutPages: Array<PageLayoutInfo> =
```

Comment:
Comment explaining this—think this holds the index and width/height for "onscreen" pages?